{
	"id": "CVE-2019-20426",
	"description": "In the Lustre file system before 2.12.3, the ptlrpc module has an out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. In the function ldlm_cancel_hpreq_check, there is no lock_count bounds check.",
	"hardware": [],
	"software": [],
	"references": [
		"http://wiki.lustre.org/Lustre_2.12.3_Changelog",
		"http://lustre.org/",
		"https://jira.whamcloud.com/browse/LU-12614",
		"https://review.whamcloud.com/#/c/36107/"
	],
	"_edited_": false
}